### DISCUSSION TITLE
using Babylonjs from leptos?

### DISCUSSION THREAD

**Discussion by ThomasCartier** [leptos-rs/leptos#2449] [Q&A]
Hi,
I wonder if we can use/drive libraries like three.js/ Babylonjs from leptos.
If yes, what would be the most efficient way since it implies Typescript/JavaScript. 
Not sure how to handle this using non js frameworks.

Thanks!

**Comment by gbj** [comment]
I'd suggest reading the chapter of the book that talks about [using JS libraries](https://book.leptos.dev/web_sys.html). If you are familiar with wasm-bindgen and have a more specific question, come back and ask with some examples of what you've tried and what does/doesn't work!
